swift/StableDiffusion/pipeline/CGImage+vImage.swift

```diff
@@ -63,58 +63,56 @@ extension CGImage {
         return cgImage
     }
     
-    public var plannerRGBShapedArray: MLShapedArray<Float32> {
-        get throws {
-            guard
-                var sourceFormat = vImage_CGImageFormat(cgImage: self),
-                var mediumFormat = vImage_CGImageFormat(
-                    bitsPerComponent: 8 * MemoryLayout<UInt8>.size,
-                    bitsPerPixel: 8 * MemoryLayout<UInt8>.size * 4,
-                    colorSpace: CGColorSpaceCreateDeviceRGB(),
-                    bitmapInfo: CGBitmapInfo(rawValue: CGImageAlphaInfo.first.rawValue)),
-                let width = vImagePixelCount(exactly: self.width),
-                let height = vImagePixelCount(exactly: self.height)
-            else {
-                throw ShapedArrayError.incorrectFormatsConvertingToShapedArray
-            }
-            
-            var sourceImageBuffer = try vImage_Buffer(cgImage: self)
-            
-            var mediumDesination = try vImage_Buffer(width: Int(width), height: Int(height), bitsPerPixel: mediumFormat.bitsPerPixel)
-            
-            let converter = vImageConverter_CreateWithCGImageFormat(
-                &sourceFormat,
-                &mediumFormat,
-                nil,
-                vImage_Flags(kvImagePrintDiagnosticsToConsole),
-                nil)
-            
-            guard let converter = converter?.takeRetainedValue() else {
-                throw ShapedArrayError.vImageConverterNotInitialized
-            }
-            
-            vImageConvert_AnyToAny(converter, &sourceImageBuffer, &mediumDesination, nil, vImage_Flags(kvImagePrintDiagnosticsToConsole))
-            
-            var destinationA = try vImage_Buffer(width: Int(width), height: Int(height), bitsPerPixel: 8 * UInt32(MemoryLayout<Float>.size))
-            var destinationR = try vImage_Buffer(width: Int(width), height: Int(height), bitsPerPixel: 8 * UInt32(MemoryLayout<Float>.size))
-            var destinationG = try vImage_Buffer(width: Int(width), height: Int(height), bitsPerPixel: 8 * UInt32(MemoryLayout<Float>.size))
-            var destinationB = try vImage_Buffer(width: Int(width), height: Int(height), bitsPerPixel: 8 * UInt32(MemoryLayout<Float>.size))
-            
-            var minFloat: [Float] = [-1.0, -1.0, -1.0, -1.0]
```

Comment:
The diff in this file looks unexpectedly large, could you please verify that the only changes are related to minFloat and maxFloat vars?